app/models/tracker.rb

```diff
@@ -161,6 +164,16 @@ def self.core_fields(trackers)
     end
   end
 
+  # Add translated attributes here and in the edit view
+  %i[name].each do |attr_name|
+    define_method :"i18n_#{attr_name}" do
+      return send(attr_name) if i18n.blank?
+
+      locale = User.current.language.to_s
+      i18n&.dig(attr_name, locale) || send(attr_name)
+    end
+  end
+
```

Comment:
@Hareramrai can we use some kind of helper or lib to generalize this property definition?

```rb
def generate_i18n_attr(attrs)
  ...
end
```

used like this: `generate_i18n_attr(%i[name])`

Reply:
yes, but I guess, we need to use `self` in the method name.

Reply:
Thanks Hare, the idea with a concern works like a charm